rnaseq_workflows/repository_dependencies.xml

```diff
@@ -4,4 +4,5 @@
   <repository toolshed="toolshed.g2.bx.psu.edu" name="trimmomatic" owner="pjbriggs" changeset_revision="f8a9a5eaca8a" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="tophat2_with_gene_annotations" owner="scottx611x" changeset_revision="015a0cc3c012" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="featurecounts" owner="yhoogstrate" changeset_revision="e04fbcc4e91a" />
+  <repository toolshed="testtoolshed.g2.bx.psu.edu" name="split_paired_collection" owner="scottx611x" changeset_revision="429cae20b148" />
```

Comment:
This tool should be published in https://toolshed.g2.bx.psu.edu/view/refinery-platform and the source code should be in https://github.com/refinery-platform/galaxy-tools.